### PR TITLE
protos for registry

### PR DESCRIPTION
**Pull request by ravishetye**
syncing with private repo this PR has other changes as well

### CODE REVIEW

**Review by conradoverta — APPROVED** [review]

**Review by evgeniilezhnin — APPROVED** [review]

File: protos/protos/public/registry/RegistryService.proto [hunk 1]
```diff
@@ -0,0 +1,379 @@
+syntax = "proto3";
+
+package ai.verta.registry;
+option go_package = "github.com/VertaAI/protos-all/protos/gen/go/protos/private/registry";
```
Comment (evgeniilezhnin) [inline protos/protos/public/registry/RegistryService.proto]:
```suggestion
option go_package = "github.com/VertaAI/modeldb/protos/gen/go/protos/public/registry";
```

should be replaced to public?

File: protos/protos/public/registry/RegistryService.proto [hunk 2]
```diff
@@ -0,0 +1,379 @@
...
+    int64 time_updated = 4;
+
+    // Description of this ``registered_model``.
+    string description = 5;
+
+    // Labels added to the registered models
+    repeated string labels = 6;
+
+    // Visibility of Registered Models
+    common.VisibilityEnum.Visibility visibility = 7;
```
Comment (evgeniilezhnin) [inline protos/protos/public/registry/RegistryService.proto:33]: There is no VisibilityEnum in CommonService

Comment (ravishetye) [reply protos/protos/public/registry/RegistryService.proto:33]: fixed

File: protos/protos/public/common/CommonService.proto [hunk 3]
```diff
@@ -69,6 +69,7 @@ message ModelDBResourceEnum {
         DATASET_VERSION = 6;
         DASHBOARD = 7;
         REPOSITORY = 8;
+        REGISTERED_MODEL = 9;
```
Comment (conradoverta) [inline protos/protos/public/common/CommonService.proto:72]: Why is this in common?

**Review by conradoverta — APPROVED** [review]
I'm assuming this is just a sync

File: protos/protos/public/common/CommonService.proto (hunk 3 above)
Comment (ravishetye) [reply protos/protos/public/common/CommonService.proto:72]: i can not figure out a good reason for it right now, but we/you did move it here right during the initial private //public restructure . https://github.com/VertaAI/modeldb/pull/736

Comment (conradoverta) [reply protos/protos/public/common/CommonService.proto:72]: Something to look at later. Not a blocker.

**Pull request merged**